samples/readme.md

```diff
@@ -3,23 +3,11 @@
 ## .NET
 
 ### Microsoft.Azure.ServiceBus
-Any samples within the `Azure.Microsoft.ServiceBus` folder target the newer .NET Standard library.
-
-#### SendSample
-
-[This sample](./DotNet/Microsoft.Azure.ServiceBus/SendSample/readme.md) shows how to write a .NET Core console application that sends messages to Service Bus.
-
-#### ReceiveSample
-
-[This sample](./DotNet/Microsoft.Azure.ServiceBus/ReceiveSample/readme.md) shows how to write a .NET Core console application that receives messages from Service Bus.
+Any samples within the `Azure.Microsoft.ServiceBus` folder target the newer .NET Standard library. Each of the sample page carries a description of the scenario covered in the sample
+under the `What will be accomplished` section.
 
 ### Microsoft.ServiceBus.Messaging
-Any samples within the `Microsoft.ServiceBus.Messaging` folder, reference the older `WindowsAzure.ServiceBus` NuGet package. These samples were moved from [this previous repository](https://github.com/Azure-Samples/azure-servicebus-messaging-samples). See the readme [here](./DotNet/Microsoft.ServiceBus.Messaging/readme.md) for more details. 
+Any samples within the `Microsoft.ServiceBus.Messaging` folder, reference the older `WindowsAzure.ServiceBus` NuGet package. These samples were moved from [this previous repository](https://github.com/Azure-Samples/azure-servicebus-messaging-samples). See the readme [here](./DotNet/Microsoft.ServiceBus.Messaging/README.md) for more details. 
```

Comment:
@nemakam `Microsoft.Azure.ServiceBus` targets .NET Full framework and .NET Standard. So that would be a bit confusing. Why not to stick to the client names?